backend/algorithms/errors.json

```diff
@@ -7343,38 +7242,27 @@
     "CHEM6701": {
         "condition": {
             "original": "12 UOC of CHEM2011, CHEM2021, CHEM2031, CHEM2839, CHEM2041 & CHEM2828<br/><br/>",
-            "processed": "12UOC of CHEM2011, CHEM2021, CHEM2031, CHEM2839, CHEM2041 & CHEM2828"
+            "processed": "12UOC of CHEM2011 && CHEM2021 && CHEM2031 && CHEM2839 && CHEM2041 && CHEM2828"
```

Comment:
Not at all intended, perhaps this sort of thing should be dealt with before commas are replaced with &&s